### DISCUSSION TITLE
Is there  plugin to highlight nodes depending on the filetype/extension?

### DISCUSSION THREAD

**Discussion by ubaldot** [lambdalisue/vim-fern#494] [Q&A]
I know that something can be achieved with (this plugin)[fern-renderer-nerdfont.vim](https://github.com/lambdalisue/fern-renderer-nerdfont.vim) but I am not interested in installing NerdFonts. I just need nodes to be highlighted based on their filetype/extension - for example .md files are yellow, .c files are blue, etc.

**Comment by lambdalisue** [comment]
Unfortunately, I don't know. Technically possible so someone already made it?

**Reply by ubaldot** [reply]
I skimmed through the plugins page but I haven’t found any. 
I think that should be a low hanging fruit, as it would be enough to associate a highlight group to each file extension and place such a mapping in ftplugin/fern.vim.

This is a bit off-topic but the same would apply to the default mappings. I saw in several .vimrc an autogroup based on FileType fern that calls a InitFunction that sets a number of buffer nmap  but I am may be just talking no-sense :) 
Did anyone considered having these mappings in ftplugin/fern.vim? :)
